hmf-common/src/main/java/com/hartwig/hmftools/common/doid/DiseaseOntology.java

```diff
@@ -203,19 +197,23 @@ private static List<DoidLogicalDefinitionAxioms> extractDoidLogicalDefinitionAxi
             doidLogicalDefinitionAxiomsChecker.check(logicalDefinitionAxiomObject);
 
             List<DoidRestriction> restrictionList = Lists.newArrayList();
```

Comment:
In the `DoidLogicalDefinitionAxioms` you made this property nullable but the implementation generates an empty list in case of null restrictions. I would suggest to map null restrictions to an actual null value in our implementation of datamodel mappping (since null restrictions may mean something different no restrictions)